debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (22.4-0ubuntu0~18.04.1ubuntu1) UNRELEASED; urgency=medium
```

Comment:
This should be 18.04.2, but doesn't matter is this will go unreleased until a new upstream snapshot.